### ISSUE TITLE
Usage with auto-approve bots like renovate-approve

### ISSUE THREAD

**Issue by roni-frantchi** [bors-ng/bors-ng#981]
We're using [Renovate](https://github.com/renovatebot/renovate) to maintain our project's dependencies.  
It has this nifty feature where once a PR is approved, it can merge it using a comment - like `bors r+`.  
Combined with [Renovate Approve](https://github.com/apps/renovate-approve)'s ability to auto-approve certain versions (like patch versions), we wanted to have all these little patch version bumps auto approved and merged.  

The problem is that the renovate-approve bot is not in `bors`'s list of approvers.  
I saw `bors` has a way to configure bots as code owners, but is there any way to add a bot to the list of pr approvers?

**Comment by notriddle** [comment]
Yeah, add the suffix `[bot]` to the user. For example, `renovate-approve[bot]`

**Comment by roni-frantchi** [comment]
Thanks @notriddle .  
I guess that can only be done by turning off the **Synchronize** option for reviewers though - as I cannot add `renovate-approve[bot]` to the list otherwise (I don't think I can add `renovate-approve[bot]` though GitHub, can I?).

**Comment by notriddle** [comment]
Yes, you would have to turn off the synchronize option to do it.

**Comment by roni-frantchi** [comment]
Thanks @notriddle .  

Guess we can keep this issue as an enhancement request to allow to add bot users when using synchronize?

**Comment by Xavientois** [comment]
We ran into this same conundrum where we want to use `renovate[bot]` to merge with bors, but we also want to synchronize the rest of the repo reviewers. This enhancement would also be useful for us.

Would the idea be to have a separate list of reviewers that can be manually configured in addition to the synced one?
